core/src/main/java/co/aikar/commands/CommandManager.java

```diff
@@ -104,7 +104,13 @@ public void registerExceptionHandler(ExceptionHandler exceptionHandler){
         exceptionHandlers.add(exceptionHandler);
     }
 
-    protected void handleUncaughtException(CommandIssuer sender, List<String> args, Exception e){
-        exceptionHandlers.forEach(handler -> handler.execute(sender, args, e));
+    protected boolean handleUncaughtException(BaseCommand scope, RegisteredCommand registeredCommand, CommandIssuer sender, List<String> args, Throwable t){
+        boolean result = false;
+        for(ExceptionHandler handler : exceptionHandlers){
+            if(handler.execute(scope,registeredCommand,sender,args,t)){
```

Comment:
i'm not sure we even need more than 1 exception handler to begin with since the manager is bound to the plugins scope.

Let's just get rid of having multiple on the manager, and make it "set default"
Update the manager to be .setDefaultExceptionHandler for method name.

Then, let's also make it so you can do new Command().setExceptionHandler(...);

Then if the command doesn't have its own set, then check if the manager has a default set